ui/public/react-globe/moon_landings.json

```diff
@@ -0,0 +1,155 @@
+[
```

Comment:
It would be cool to add the Intuitive Machines Launch site and the Indian lander as well

Reply:
Also maybe an easter egg for "moondao's future lunar base"